### PR TITLE
Bookstore: initialize

### PR DESCRIPTION
**Pull request by Young-Gaius**
This pull request adds the necessary components and functionality to meet the project requirements. Here's a summary of the changes:

Created a /components directory to store reusable components.
Installed React Router V6 and set up routing inside .
Added two routes: "/" and "/categories", each rendering a respective page component.
Implemented individual book components with the remove button.
Created a book component to render individual books.
Added an aadBook component to create new books.
Updated the design of the bookstore to replace the category input with an author input.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Books from './Books';
+import AddBook from './AddBook';
+
+const BooksPage = () => (
+  <>
+    <Books />
```
Comment (Qoosim) [inline src/components/Book.js]: - [ ] Great job for initializing your component. Hhowever, there's little stuff to get correctly here. This BooksPage should render list of books using your Books component. It requires props to be passed and use that props in the Books component to render the value of the props on the browser. It should look something similar to this :point_down: 

```
const BooksPage = () => (
   <>
      <Books title="Book One" author="Author One" />
      <AddBook />
    </>  
)
```

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        <h3>
+          Title:
+          {'me '}
+          {title}
+        </h3>
+        <h3>
+          By:
+          {' me'}
+          {author}
+        </h3>
```
Comment (Qoosim) [inline src/components/Books.js]: - [ ] What is needed here is to use the props and the values would be displayed. Something like this :point_down: 

```
   <h3>{title}</h3>
     
   <h3>Byt: {author}</h3>
```

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Young-Gaius, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — APPROVED** [review]
# Hello @Young-Gaius       😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯


- Good implementation of the required changes👍 
- well-organized file structure✔ 👍
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**